src/bootstrap/util.rs

```diff
@@ -45,6 +45,7 @@ pub fn libdir(target: TargetSelection) -> &'static str {
 }
 
 /// Adds a list of lookup paths to `cmd`'s dynamic library lookup path.
+/// If The dylib_path_par is already set for this cmd, the old value will be overwritten!
```

Comment:
Can we fix it to not do this? Overwriting the old value seems very confusing.

Reply:
(a) this is pre-existing, I'd rather not touch this since I have no idea what changing this will break,
(b) it requires an unstable feature (https://github.com/rust-lang/rust/issues/44434), which AFAIK bootstrap tries to avoid.